demo/demo.cmake

```diff
@@ -23,10 +23,9 @@ if(sdl2_FOUND)
     DEMO_RESOURCE_DIR="${CMAKE_CURRENT_SOURCE_DIR}/demo/resources"
     )
 
-  target_include_directories(CsPaintDemo PRIVATE ${SDL2_INCLUDE_DIRS}
+  target_include_directories(CsPaintDemo PRIVATE SDL2::SDL2
```

Comment:
The FindSDL.cmake script on Debian does not populate any target named SDL2::SDL2, so this change breaks the build on my machine. Not sure the best way to move forward on this. Any ideas?

Reply:
Yes, I have a good idea on solving this, but need to test it. I'll update the PR asap :)

Reply:
Updated the code to fix this issue. Quick Docker-based verification with Debian 10 verifies the change should do what I expect. Will need a CI run to verify fully.